lib/winston/transports/http.js

```diff
@@ -177,13 +177,17 @@ module.exports = class Http extends TransportStream {
     delete options.path;
 
     // Prepare options for outgoing HTTP request
+    const headers = Object.assign({}, this.headers);
```

Comment:
why do you need a new object here, can you modify this.headers directly?

Reply:
I need a new object because I imagine some requests may use bearer, some may not. 
The case I want to avoid:
1. request 1 has auth in object and has bearer added
2. request 2 has no auth in object, so no bearer should be added
-> if i were to not create a clone, request 2 would also have bearer added